exampleCourse/questions/demo/autograder/r/createRObject/README.md

```diff
@@ -0,0 +1,13 @@
+
+## RStudio Workspace
```

Comment:
We don't currently include `README.md` files in example course questions; the question itself should contain any explanatory information.

Reply:
Removed in b9c42abce.  It was just meant to provide context for those looking at the code of questions.